packages/middleware/package.json

```diff
@@ -25,7 +25,8 @@
     "cors": "^2.8.5",
     "express": "^4.18.1",
     "helmet": "^5.1.1",
-    "@godaddy/terminus": "^4.12.1"
+    "@godaddy/terminus": "^4.12.1",
+    "@vue-storefront/logger": "1.0.0-rc.3"
```

Comment:
I know it is very picky from my side, but can we move our packages to the top of deps list? Also, I think that the consola can be removed at this stage?